README.md

```
@@ -1,8 +1,3 @@
 # Clone Tabnews
 
-Recriação do site [tabnews.com.br](https://tabnews.com.br) como parte do curso oferecido pelo [curso.dev](https://curso.dev), desenvolvido por Felipe Deschamps.
-
-## Tecnologias Utilizadas
-
-- React
-- Next.js
+Recriação do site [tabnews.com.br](https://tabnews.com.br) como parte do curso oferecido pelo [curso.dev](https://curso.dev), desenvolvido por Felipe Deschamps.
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)